graphql_compiler/query_pagination/query_parameterizer.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from collections import namedtuple
+
+
+RESERVED_PARAMETER_PREFIX = '_paged_'
+
+ParameterizedPaginationQueries = namedtuple(
+    'ParameterizedPaginationQueries',
+    (
+        'next_page_query',          # Document, AST of query that will return the next page of
+                                    # results when hydrated with pagination parameters.
+        'continuation_query',       # Document, AST of query that will return the remainder of
+                                    # results when hydrated with pagination parameters.
+        'pagination_filters',       # List[PaginationFilter], filters usable for pagination.
+        'user_parameters',          # dict, parameters that the user has defined for other filters.
+    ),
+)
+
+PaginationFilter = namedtuple(
+    'PaginationFilter',
+    (
+        'vertex_field',                 # str, vertex class to which the property field belongs to.
```

Comment:
`vertex_class` and `vertex_field` are different things. The name of a `vertex_field` is usually a name of an edge class.